pkg/virt-handler/vm.go

```diff
@@ -605,6 +606,14 @@ func (d *VirtualMachineController) updateVMIStatus(vmi *v1.VirtualMachineInstanc
 				// Remove the interface from domainInterfaceStatusByMac to mark it as handled
 				if interfaceStatus, exists := domainInterfaceStatusByMac[interfaceMAC]; exists {
 					newInterface.InterfaceName = interfaceStatus.InterfaceName
+
+					// Qemu guest reports CIDR, only an IP address should be passed to the status
```

Comment:
I would expect the name of the function to say this without the need to add comments here.